README.md

```diff
@@ -4,7 +4,7 @@
 
 A collection of source code generators for [Java][java].
 
-## Auto‽
```

Comment:
I think this interrobang was deliberate. But it's also potentially confusing. If you just delete it then this section heading is the same as the previous one, which is not great. Maybe change it to something completely different, like `## Overview`.